Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeysProjection+consistentReadQuery.swift

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+//  DynamoDBCompositePrimaryKeysProjection+consistentReadQuery.swift
+//  SmokeDynamoDB
+//
+
+import Foundation
+import SmokeHTTPClient
+import DynamoDBModel
+import NIO
+
+public extension DynamoDBCompositePrimaryKeysProjection {
```

Comment:
It doesn't make sense to have a consistentRead parameter on a Projection as a projection is always going to be a GSI.

Reply:
It's good to know that projection only used for GSI. I will remove consistentRead parameter and set it to false inside the query function.